src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+
+        <label for="surname">
+          Surname:  
+          <input 
+            type="text"
+            class="surname" 
+            name="surname" 
+            required
+          >
+        </label>
```

Comment:
ok, now you use `label`s and it is good, but they don't work. You should set the same value to `for` and to `input`'s `id`. 
or just remove `for `if the `label` wraps the `input`

Reply:
fix it everywhere